controllers/SessaoTreinoController.js

```diff
@@ -2,8 +2,21 @@ const SessaoTreinoService = require('../services/SessaoTreinoService');
 const ExercicioService = require('../services/ExercicioService');
 const crypto = require('crypto')
 
+function makeid(length) {
+    let result = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+    const charactersLength = characters.length;
+    let counter = 0;
+    while (counter < length) {
+      result += characters.charAt(Math.floor(Math.random() * charactersLength));
+      counter += 1;
+    }
+    return result;
+}
```

Comment:
Função para gerar nome da variavel aleatoria 
https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript